tox.ini

```diff
@@ -1,11 +1,11 @@
 [tox]
 
-
 envlist = py-sqlalchemy
 
 SQLA_REPO = {env:SQLA_REPO:git+https://github.com/sqlalchemy/sqlalchemy.git}
 
 [testenv]
+basepython = py311
```

Comment:
**Michael Bayer** (zzzeek) wrote:

ah.   this breaks things can we remove this please

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/5455

Reply:
My bad, that shouldn't have been left in